src/main/java/uk/gov/hmcts/reform/civil/notification/handlers/createclaimspecafterpayment/CaseTakenOfflineForSpecAllPartiesEmailGenerator.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.civil.notification.handlers.createclaimspecafterpayment;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.civil.notification.handlers.AllPartiesEmailGenerator;
+
+import java.util.List;
+
+@Component
+public class CaseTakenOfflineForSpecAllPartiesEmailGenerator extends AllPartiesEmailGenerator {
```

Comment:
can we split these files into different folders for ex-  https://github.com/hmcts/civil-service/tree/master/src/main/java/uk/gov/hmcts/reform/civil/notification/handlers/defendantresponse/unspec/offline/otherresponses